src/AssetsPage/AssetsTable/index.jsx

```diff
@@ -18,16 +18,57 @@ class AssetsTable extends React.Component {
       modalOpen: false,
       assetToDelete: {},
       elementToFocusOnModalClose: {},
+      tableColumns: [
+        // keyed object? or just search through the object when I need it? performance?
+        {
+          // label: this.renderTableColumnButton('Name', true, 'descending'),
+          label: 'Name',
+          key: 'display_name',
+          sortable: true,
+          // redundancy :(
+          sortDirection: 'descending',
+        },
+        {
+          // label: this.renderTableColumnButton('Type', true, 'none'),
+          label: 'Type',
+          key: 'content_type',
+          sortable: true,
+          sortDirection: 'none',
+        },
+        {
+          // label: this.renderTableColumnButton('Date Added', true, ''),
+          label: 'Date Added',
+          key: 'date_added',
+          sortable: true,
+          sortDirection: 'none',
+        },
+        {
+          // label: this.renderTableColumnButton('Delete Asset', false, ''),
+          label: 'Delete Asset',
+          key: 'delete_asset',
+          sortable: false,
+        },
+      ],
+      sortState: {
+        // should this be dynamic instead of hardcoded? first sortable element of the columns? how to do that?
```

Comment:
This is assuming that the default initial state is to sort the first sortable element of the column in descending order.

Reply:
I could see date being the default sorting as well, but that's probably a question to be determined by product later on in this process.